python/python/Ice/UserException.py

```diff
@@ -5,12 +5,5 @@
 class UserException(Exception):
     """The base class for all user-defined exceptions."""
 
-    def ice_getSlicedData(self):
```

Comment:
This method was already removed from UserException in other mappings.

Reply:
Do we need `__module__` and `__class__` below? UserException is logically abstract and should never be instantiated by the C++ code.

Reply:
It is still required because the C++ code looks up the type when it needs to check if an exception is a UserException.

```
IcePy/ObjectAdapter.cpp
153:        PyObject* userExceptionType = lookupType("Ice.UserException");
238:        PyObject* userExceptionType = lookupType("Ice.UserException");

IcePy/Operation.cpp
2464:            PyObject* userExceptionType = lookupType("Ice.UserException");

IcePy/Util.cpp
298:    PyObject* userExceptionType = lookupType("Ice.UserException");
```

Of course, we can also fix the extension to use `Ice.UserException.UserException`

Reply:
`__class__` can be removed either way